CONTRIBUTING.md

```diff
@@ -643,6 +643,29 @@ Few notes on GPU builds:
 3. We don't need NVIDIA® drivers for building, but we need them for running simulations
 4. Only Linux platforms are supported
 
+Qiskit Aer now supports cuQuantum optimized Quantum computing APIs from NVIDIA®.
+cuStateVec APIs can be exploited to accelerate statevector, density_matrix and unitary methods.
+Because cuQuantum is beta version currently, some of the operations are not accelerated by cuStateVec.
+
+To build Qiskit Aer with cuStateVec support, please set the path to cuQuantum root directory to CUSTATEVEC_ROOT as following.
+
+For example,
+
+    qiskit-aer$ python ./setup.py bdist_wheel -- -DAER_THRUST_BACKEND=CUDA -DCUSTATEVEC_ROOT=path_to_cuQuantum
+
+To run with cuStateVec, set `device='GPU'` to AerSimulator option and cuStateVec is enabled 
+if the number of qubits of input circuit is equal or greater than 22 qubits by default.
+This threshold can be modified by setting `cuStateVec_threshold` option.
+By setting `cuStateVec_enable=False` to disable using cuStateVec. 
+Following example shows how you accelerate 10 or more qubits simulations using cuStateVec.
+
+```
+sim = AerSimulator(method='statevector', device='GPU')
+results = execute(circuit,sim,cuStateVec_enable=True,cuStateVec_threshold=10).result()
+```
+
```

Comment:
It is better to explain other methods (not only `statevector`) are also supported.

Reply:
added